exporter/views.py

```diff
@@ -26,6 +30,12 @@ def exporter_start(request):
 
 @csrf_exempt
 def wiper_start(request):
+    """
+    Plans (send messages to a worker) the deletion of exported collection
+    (all generated files in export folders).
+
+    Expects {"collection_id": <id>} in request body.
```

Comment:
"all generated files in export folders" sounds like literally all files in the folder (i.e. not just for one collection, but all collections). I assume the wiper only affects one collection.

```suggestion
    Adds a message to a queue to delete the files exported from a collection.

    Expects ``{"collection_id": <id>}`` in the request body.
```

Reply:
Again, reading the code, I'm pretty sure spider and job_id need to be set. That's what the wiper expects.